src/docker/Dockerfile

```diff
@@ -52,43 +52,31 @@ RUN apt-get update && apt-get install -y \
     automake autoconf
 
 # Git 2.0
-RUN apt-get install -y software-properties-common python-software-properties
+RUN apt-get update && apt-get install -y \
```

Comment:
@amihaiemil I'm just suggesting to make this Docker file shorter. Instead of having many sections, you can have just three, e.g.
```dockerfile
RUN apt-get update && apt-get install -y software-properties-common \
  && apt-add-repository ppa:brightbox/ruby-ng

RUN apt-get update && apt-get install -y \
  openjdk-7-jdk \
  git \
  s3cmd \
  build-essential \
  libxml2-utils \
  libmagic-dev=1:5.14-2ubuntu3.3 \
  zlib1g-dev=1:1.2.8.dfsg-1ubuntu1 \
  ruby2.2=2.2.4-1bbox1~trusty1 \
  ruby2.2-dev=2.2.4-1bbox1~trusty1 \
  && rm -rf /var/lib/apt/lists/*

RUN gem update && gem install \
  nokogiri:1.6.7.2 \
  bundler:1.11.2 \
  pdd:0.15.4 \
  est:0.3.1
```
If you don't have Docker experience that's fine, you've already spend time here, you can puzzle out the rest